simglucose/simulation/scenario.py

```diff
@@ -40,11 +40,14 @@ def __init__(self, start_time=None, scenario=None):
         self.scenario = scenario
 
     def get_action(self, t):
-        times, actions = tuple(zip(*self.scenario))
-        times2compare = [parseTime(time, self.start_time) for time in times]
-        if t in times2compare:
-            idx = times2compare.index(t)
-            return Action(meal=actions[idx])
+        if self.scenario:
```

Comment:
Thank you so much for your pull request! I am so sorry that I didn't notice your PR until now. 

To protect the empty case, can you do it the other way to make the code cleaner. I mean 

···
if not self.scenario:
  return Action(meal=0)

# Then the original code
···